inference-engine/src/transformations/src/transformations/common_optimizations/relu_fake_quantize_fusion.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/relu_fake_quantize_fusion.hpp"
+#include "transformations/utils/utils.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ReluFakeQuantizeFusion, "ReluFakeQuantizeFusion", 0);
+
+ngraph::pass::ReluFakeQuantizeFusion::ReluFakeQuantizeFusion() {
+    auto data_pattern = ngraph::pattern::any_input();
+    auto relu_pattern = ngraph::pattern::wrap_type<opset5::Relu>({data_pattern}, pattern::consumers_count(1));
+    auto input_low_pattern = ngraph::pattern::wrap_type<opset5::Constant>();
+    auto fq_pattern = ngraph::pattern::wrap_type<opset5::FakeQuantize>({relu_pattern, input_low_pattern,
+                                                                        ngraph::pattern::wrap_type<opset5::Constant>(),
+                                                                        ngraph::pattern::wrap_type<opset5::Constant>(),
+                                                                        ngraph::pattern::wrap_type<opset5::Constant>()});
```

Comment:
If we require only `input_low` to be a Constant then we can use `ngraph::pattern::any_input();` for other inputs.
```suggestion
    auto fq_pattern = ngraph::pattern::wrap_type<opset5::FakeQuantize>({relu_pattern, input_low_pattern,
                                                                        ngraph::pattern::any_input(),
                                                                        ngraph::pattern::any_input(),
                                                                        ngraph::pattern::any_input()});
```